tests/security_and_privacy/test_phishing_and_malware_warnings.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+from modules.browser_object_tabbar import TabBar
+
+
+@pytest.fixture()
+def add_prefs():
+    return [
+        ("browser.safebrowsing.malware.enabled", True),
+        ("browser.safebrowsing.phishing.enabled", True),
+    ]
+
+
+def get_error_title(driver, url):
+    """
+    Navigate to the given URL and return the error title text.
+    """
+    driver.get(url)
+    error_title_element = WebDriverWait(driver, 10).until(
+        EC.presence_of_element_located((By.ID, "errorTitleText"))
+    )
+    return error_title_element.get_attribute("innerText")
+
+
+def test_phishing_and_malware_warning_errors(driver: Firefox):
+    """
+    C1901393: - This tests that when a user accesses websites suspected of phishing or malware, a warning page is
+    displayed.
+    """
+    # Create the object
+    tabs = TabBar(driver)
+
+    # Test phishing warning page
+    phishing_url = "http://www.itisatrap.org/firefox/its-a-trap.html"
```

Comment:
Can we move constants like URLs to the top in the format of `PHISHING_URL = "http://www.itisatrap.org/firefox/its-a-trap.html"`?

Reply:
Sure